src/gatling/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/gatling/simulations/BookingsReportGenerationTimeSimulation.kt

```diff
@@ -140,6 +153,11 @@ class BookingsReportGenerationTimeSimulation : Simulation() {
       setupBooking.injectOpen(
         constantUsersPerSec(20.0).during(60.seconds.toJavaDuration()),
       )
+        .andThen(
+          bookingsSearchJourney.injectOpen(
```

Comment:
We're currently trying to figure out how we write assertions for the other tests so that these fail if any request takes longer than say 5 seconds.

We had a quick test and if we follow this https://gatling.io/docs/gatling/reference/current/core/assertions/ we get global() undefined or similar. Probably not importing something correctly I suspect. I appreciate this probably wasn't in the original ask but is something we need to figure out before the test will notify anyone of a problem. [Thread here if you have any ideas or can make it work](https://mojdt.slack.com/archives/C03K0HB0LBE/p1702573213708589?thread_ts=1702474801.256889&cid=C03K0HB0LBE)

Reply:
I'll take a look once i've finished my current ticket  👍

Reply:
I've added some global() assertions in for both simulations and i can see the build failing if they're sufficient low. They're very high at the moment to get tests running in dev after a few runs. I guess we at least we have a high water mark we can start pulling down